### PR TITLE
Add targeted vp example.

### PR DESCRIPTION
**Pull request by dlongley**
Addresses #2.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-imp-guide/pull/39.html" title="Last updated on Aug 14, 2019, 4:43 PM UTC (1ef403f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-imp-guide/39/ff354eb...1ef403f.html" title="Last updated on Aug 14, 2019, 4:43 PM UTC (1ef403f)">Diff</a>

### CODE REVIEW

**Review by brentzundel — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -305,6 +305,48 @@ <h2>Disputes</h2>
...
+using a <a>verifiable presentation</a>. A <a>verifiable presentation</a> can
+be targeted to a specific <a>verifier</a> by using a Linked Data Proof
+that includes a <code>domain</code> and <code>challenge</code>. This also
+helps prevent a <a>verifier</a> from reusing a <a>verifiable presentation</a>
+as their own.
+      </p>
+
+      <p>
+The following <a>verifiable presentation</a> is for authenticating to
+a website <code>https://example.com</code>. Note that the <code>domain</code>
```
Comment (TallTed) [inline index.html]: `website <code>`
-> `website, <code>`
(add comma)

File: index.html [hunk 2]
```diff
@@ -305,6 +305,48 @@ <h2>Disputes</h2>
...
+value can be any string or URI and the <code>challenge</code> should be a
```
Comment (TallTed) [inline index.html]: `URI and`
-> `URI, and`
(add comma)

File: index.html [hunk 3]
```diff
@@ -305,6 +305,48 @@ <h2>Disputes</h2>
...
+      </p>
+
+
+      <pre class="example nohighlight" title="A targeted verifiable presentation">
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1"
+  ],
+  "type": "VerifiablePresentation,
+  "verifiableCredential": { ... },
```
Comment (TallTed) [inline index.html]: needs CSS magic on `...` here and below (and I think `..` in `jws` should be `...`)

Comment (dlongley) [reply index.html]: Possibly yes to the CSS magic ... but the `..` (two dots) is correct... that's actually part of the JWS.

**Comment by David-Chadwick** [comment]
We already have the concept of targetted VCs in JWT proofed VPs. It is the aud claim. Therefore we should enhance this text to say how the same concept can be implemented with JWTs.
@dlongley would you like me to produce a PR for this or will you?

**Comment by dlongley** [comment]
@David-Chadwick -- Please add that PR on top of this one, thanks!

**Comment by David-Chadwick** [comment]
Will do. As soon as your PR is committed then I can add a subsection 6.1 Audience

**Comment by David-Chadwick** [comment]
Because I don't know how to add a PR to an existing PR :-)

File: index.html (hunk 3 above)
Comment (dlongley) [reply index.html]: CSS magic added.

**Comment by dlongley** [comment]
@David-Chadwick,

> Will do. As soon as your PR is committed then I can add a subsection 6.1 Audience
> Because I don't know how to add a PR to an existing PR :-)

That's fine :). Btw, the way you do it is to just branch off of the branch for this PR.

**Review by TallTed — CHANGES_REQUESTED** [review]
I suggest one (moderately large) change, breaking this paragraph and reordering its content --
```
      <p>
The following <a>verifiable presentation</a> is for authenticating to
a website, <code>https://example.com</code>. Note that the <code>domain</code>
value can be any string or URI, and the <code>challenge</code> should be a
randomly generated string.
      </p>
```
-- making it --
```
      <p>
The <code>domain</code> value can be any string or URI, and the <code>challenge</code>
should be a randomly generated string.
      </p>

      <p>
The following sample <a>verifiable presentation</a> is for authenticating to
a website, <code>https://example.com</code>.
      </p>
```

**Comment by dlongley** [comment]
@TallTed -- changed. Look good?

**Review by TallTed — APPROVED** [review]

**Pull request merged**